src/main/scala/microsites/layouts/DocsLayout.scala

```diff
@@ -155,4 +156,21 @@ class DocsLayout(config: MicrositeSettings) extends Layout(config) {
       (if (config.micrositeKazariSettings.micrositeKazariEnabled)
          List(script(src := "{{ site.baseurl }}/js/kazari.js"))
        else List.empty[TypedTag[String]])
+
+  def editButton: Seq[TypedTag[String]] =
+    config.templateTexts.editButton match {
+      case Some(text) =>
+        Seq(
+          div(
+            cls := "edit-button",
+            a(
+              href := config.gitSiteUrl,
+              href := "/edit/master/modules/docs/src/main/tut/{{ page.path }}",
```

Comment:
Although maybe we can this baseUrl to a new setting, for example, in the sbt-microsites case, this would be something like this:

`/edit/master/docs/src/main/tut/docs/{{ page.path }}`

Hence, my suggestion would to create a product for the new sbt setting:

```scala
micrositeEditButton: SettingKey[Option[EditButtonSettings]]

// ... and

case class EditButtonSettings(text: String, baseDocsPath: String)
```

Thoughts?

Reply:
I think that's a good idea. i was debating if I should make that go to the edit view of the page or not, this way users could decide for themselves.

Reply:
ah and yes I realized I was basing my path off a project with modules, so the path could vary depending on that as well.